lib/static/css/main.css

```diff
@@ -752,10 +752,14 @@ blockquote {
   margin-right: 10px;
 }
 
+.fixedHeaderContainer header .headerTitle {
+  font-size: 1.1em;
+  margin: 0;
+}
+
 .fixedHeaderContainer header .headerTitleWithLogo {
   display: block;
-  font-size: 20px;
-  font-weight: 400;
+  font-size: 1.1em;
```

Comment:
Assuming default base pixel size of 16px, do we want `1.250em` -> 20px (same to old value) instead of  `1.1em` -> 17px

Reply:
Ok I'll change to that. I felt that 1.1em looks nicer 😂